iothub/service/src/IotHubServiceClient.cs

```diff
@@ -161,6 +161,14 @@ public IotHubServiceClient(string hostName, AzureSasCredential credential, IotHu
         /// <seealso href="https://docs.microsoft.com/azure/iot-develop/concepts-digital-twin"/>
         public DigitalTwinsClient DigitalTwins { get; protected set; }
 
+        /// <summary>
+        /// Subclient of <see cref="IotHubServiceClient"/> that handles getting, updating and replacing device twin and module twin.
```

Comment:
nit: use an oxford comma

```suggestion
        /// Subclient of <see cref="IotHubServiceClient"/> that handles getting, updating, and replacing device and module twins.
```